packages/components/src/components/SquiggleViewer/ValueWithContextViewer.tsx

```diff
@@ -68,15 +68,15 @@ const WithComment: FC<PropsWithChildren<Props>> = ({ value, children }) => {
   const commentPosition = tagsWithTopPosition.has(value.tag) ? "top" : "bottom";
 
   const commentEl = (
-    <ReactMarkdown
+    <div
       className={clsx(
-        "prose max-w-4xl text-sm text-stone-600",
+        "max-w-4xl text-sm text-stone-600",
```

Comment:
`text-sm text-stone-600` belong in `<MarkdownViewer>`, not here (overriding these outside of the component is fine if the component is mostly the plain text, but markdown has headers, so `<MarkdownViewer>` should be aware of its default font size to be self-contained).

Upd: I see in another component you use `text-xs`; this is even more of a reason to configure it inside `MarkdownViewer`, with a custom prop such as `size: "normal" | "small"`. This might be out of scope of this PR, though, I see you haven't tuned headers yet.

Reply:
I tried changing this, curious if you have quick takes.